lib/options-extractors.js

```diff
@@ -43,13 +43,9 @@ function exportServiceOptions(options) {
     fastifyPluginOptions.prefix = prefix
   }
 
-  if (envVariables) {
-    for (const key in envVariables) {
-      if ({}.hasOwnProperty.call(envVariables, key)) {
-        fastifyPluginOptions[key] = envVariables[key]
-      }
-    }
-  }
+  Object
```

Comment:
Why this change?

Reply:
I thought that iterate an array is faster than call hasOwnProperty for each entry

Reply:
It is not so true. In this way, the object is also converted in array. And the check is only a startup, so the performance are not so important.